_maps/map_files/Blueshift/Blueshift.dmm

```diff
@@ -3048,6 +3048,9 @@
 /area/station/science/genetics)
 "aFb" = (
 /obj/structure/reagent_dispensers/water_cooler,
+/obj/machinery/requests_console/directional/north{
+	name = "Telecomms Requests Console"
```

Comment:
This doesn't look like it's set to let them send announcements using it?

Reply:
Alright, should be fixed along with the other maps

Reply:
What is the desk name when it announces? I forgot if it goes by the name of the request console but like how QM announcement is "Quartermasters Desk", telecoms admin is generally "Telecomms Admin" announcements.

Reply:
yeah it's the `department` var